app/views/contexts/view_preview_questions_context.py

```diff
@@ -0,0 +1,26 @@
+from flask import url_for
+
+from app.data_models import QuestionnaireStore
+from app.questionnaire.questionnaire_schema import QuestionnaireSchema
+from app.views.contexts.preview_context import PreviewContext
+
+
+def build_view_preview_questions_context(
```

Comment:
This needs to either be used for pdf / preview or it needs to be removed. It is only used in the PDF endpoint now.

Reply:
I've just removed this module/method altogether.